tests/workflows/snvphylnfc.nf.test

```diff
@@ -66,4 +66,66 @@ nextflow_workflow {
 
     }
 
+    test("selecting reference by sample id") {
```

Comment:
Is it possible to also have a test case where no reference genome is selected? This should have `assert workflow.failed` and you could then have `assert workflow.errorReport.contains("....")` to make sure the correct error message shows up (from https://www.nf-test.com/docs/testcases/nextflow_workflow/).

Reply:
Add in 708c420ebf8edce68f4e290a2c4e5310989540fa